ProcessMaker/processmaker
# Create patch v2: 4.3.0+patch-2-les-schwab

## Issue & Reproduction Steps
This PR will represent the changes necessary to create patch v2 for v4.3.0 for a ProcessMaker client as defined in [RM-1534](https://processmaker.atlassian.net/browse/RM-1534). This is to ensure prior changes will be included in the new patch when applied.

## Solution
- Merged `bugfix/FOUR-7193` into `4.3.0+patch-2-les-schwab`

## How to Test
- Ensure process request permissions queries are still optimized and performant
- Test to make sure the newer version of `processmaker/docker-executor-node-ssr` works correctly, especially when rendering many screens in parallel. 

## Related Tickets & Packages
- HELP ticket: https://processmaker.atlassian.net/browse/PM4EH-381
- Original patch request: https://processmaker.atlassian.net/browse/RM-1534
- New patch request: https://processmaker.atlassian.net/browse/RM-1538
- Changes made to be included in new patch: https://processmaker.atlassian.net/browse/FOUR-7193

## Code Review Checklist

## CircleCi
ci:deploy

[RM-1534]: https://processmaker.atlassian.net/browse/RM-1534?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
